src/wasm/jump-table-assembler.cc

```diff
@@ -325,7 +325,8 @@ bool JumpTableAssembler::EmitJumpSlot(Address target) {
 }
 
 void JumpTableAssembler::EmitFarJumpSlot(Address target) {
-  JumpToInstructionStream(target);
+  li(kOffHeapTrampolineRegister, Operand(target), ADDRESS_LOAD);
```

Comment:
Is this change safe and correct for other targets as well?

Reply:
JumpTableAssembler doesn't support growbuffer, but constatntpool need it.

Reply:
Ok, I see MIPS gets the same results as before.